satpy/multiscene.py

```diff
@@ -58,7 +58,9 @@ def timeseries(datasets):
     for ds in datasets:
         if 'time' not in ds.dims:
             tmp = ds.expand_dims("time")
-            tmp.coords["time"] = pd.DatetimeIndex([ds.attrs["start_time"]])
+            tmp.coords["time"] = pd.DatetimeIndex([ds.attrs.get('start_time', np.datetime64('NaT'))])
```

Comment:
I don't meet the situation `start_time` doesn't exist. Because I find @djhoese didn't add the `end_time` in the test case, I added this one. Maybe, we don't need to consider the situation for `start_time`?

Reply:
I would say `start_time` should always exist. `end_time` I am less sure about.

Reply:
Actually, in the case of end_time not being there, you should set it to start_time.

Reply:
Updated.